TransferMethod/Sources/View/Widget/TextWidget.swift

```diff
@@ -107,41 +107,34 @@ class TextWidget: AbstractWidget {
         if let pattern = pattern {
             let currentText = getTextForPatternCharacter(PatternCharacter.lettersAndNumbersPatternCharacter.rawValue,
                                                          inputText)
-            var currentTextIndex = inputText.startIndex
             var finalText = ""
+            var index = CurrentIndex(textIndex: inputText.startIndex, patternIndex: pattern.startIndex)
```

Comment:
Maybe call this `currentIndex` or something more appropriate

Reply:
done